plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PineryIUSValue.java

```diff
@@ -140,6 +152,7 @@ public boolean equals(Object o) {
         && project.equals(that.project)
         && rin == that.rin
         && run_status.equals(that.run_status)
+        && sequencing_kit.equals(that.sequencing_kit)
```

Comment:
Could be null

Reply:
Thanks. I've updated them to be Optionals to account for that.